internals/example-app/page.go

```diff
@@ -0,0 +1,94 @@
+package example_app
+
+const page = `
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<meta name="description" content="">
+	<meta name="author" content="">
+	<link rel="icon" href="/docs/4.0/assets/img/favicons/favicon.ico">
+
+	<title>SecretHub Example App</title>
+
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.min.css" integrity="sha256-zmfNZmXoNWBMemUOo1XUGFfc0ihGGLYdgtJS3KCr/l0=" crossorigin="anonymous" />
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
+
+	<style>
+		html,
+		body {
+			height: 100%;
+		}
+
+		body {
+			display: -ms-flexbox;
+			display: -webkit-box;
+			display: flex;
+			-ms-flex-align: center;
+			-ms-flex-pack: center;
+			-webkit-box-align: center;
+			align-items: center;
+			-webkit-box-pack: center;
+			justify-content: center;
+			padding-top: 40px;
+			padding-bottom: 40px;
+			background-color: #f5f5f5;
+		}
+
+		.container {
+			width: 100%;
+			max-width: 400px;
+			padding: 15px;
+			margin: 0 auto;
+		}
+	</style>
+
+	<script>
+		function processResult(status, icon, color, result){
+            $("#status").html(status);
+            $("#animation").attr("class", "fas fa-"+icon+" fa-5x");
+            $("#animation").css("color", color);
+            if(result !== "") {
+                $(".result-container").attr("hidden",false);
+                $("#result").html(result);
+			}
+		}
+
+		function processError(message){
+            processResult("An error occurred!", "times", "red", message);
+		}
+
+		$(function(){
+		    $.get("http://127.0.0.1:{{.}}/api", {}, function(res, status, xhr){
+		        if(xhr.status === 200) {
+                    processResult("Successfully connected to https://demo.secrethub.io/api!", "check", "green", res);
```

Comment:
What's this `https://demo.secrethub.io/api`?

Reply:
It's a custom API that's in the making